Presentation.Web/Controllers/API/V2/External/ItSystemUsages/ItSystemUsageV2Controller.cs

```diff
@@ -143,6 +143,28 @@ public IHttpActionResult GetItSystemUsagePermissions([NonEmptyGuid] Guid systemU
                 .Match(Ok, FromOperationError);
         }
 
+        /// <summary>
+        /// Returns the permissions of the authenticated client for the IT-System usage resources collection in the context of an organization (IT-System usage permissions in a specific Organization)
+        /// </summary>
+        /// <param name="organizationUuid">UUID of the organization</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("permissions")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ResourceCollectionPermissionsResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetItSystemUsageCollectionPermissions([NonEmptyGuid] Guid organizationUuid)
```

Comment:
add the `[Required]` attribute. Also do that to the other collection permission endpoints from other PR's and the one you just merged :-)